### ISSUE TITLE
Wind particles are at negative Z value

### ISSUE THREAD

**Issue by Alasdairbugs** [SuperTux/supertux#3144]
### SuperTux Version

0.6.3-1505-gc2e6b766d

### System Information

win 11 64 bit

### Expected Behavior

wind particles should not be at this low of a Z value i think

because its an object, it should default to 50

### Actual Behavior

its at below -50

### Steps To Reproduce Actual Behavior

place some tiles at z50

place wind, set to fancy particles. interacts with badguys, objects. observe.

### Guidelines For Reporting Issues

- [X] I have read <https://github.com/SuperTux/supertux/blob/master/CONTRIBUTING.md#bug-reports>.
- [X] I have verified this isn't an issue that's already been reported.
- [X] I have verified this isn't a [discussion](https://github.com/SuperTux/supertux/discussions), or an issue about a crash or a feature request, but rather an actual bug ─ that is, the game did something not intended.
- [X] I have verified this issue is **not** about wrong translations (use Transifex for those), or anything unsupported (e.g. third-party add-ons).
- [X] In this report, I have only included details about **one** (1) bug.
- [X] If I make a mistake while submitting this report, I agree to use the "Edit" feature to correct it, instead of closing this issue and opening a new one.

**Comment by MatusGuy** [comment]
Odd, code says it's at layer 50.

https://github.com/SuperTux/supertux/blob/master/src%2Fobject%2Fwind.cpp#L127

**Comment by tobbi** [comment]
No, that's for the editor image. Actual wind particles are at `LAYER_BACKGROUNDTILES - 1`, aka `-99`. 

Where do we want that?

**Comment by tobbi** [comment]
How about making it configurable?

**Comment by tobbi** [comment]
Please test the PR above.

**Comment by tobbi** [comment]
🥳 🤵
